daemon/cmd/datapath.go

```diff
@@ -536,7 +536,7 @@ func setupRouteToVtepCidr() error {
 	for _, rt := range routes {
 		log.WithFields(logrus.Fields{
 			logfields.IPAddr: rt.Dst.String(),
-		}).Info("VTEP route")
+		}).Debug("VTEP route")
```

Comment:
While we're at it, if this debug log will stay in then can we clarify what it's actually doing so the log makes some sense?

```suggestion
		}).Debug("Configuring VTEP route in the kernel")
```

Reply:
thanks for asking the question, the log line was added while I was developing the feature to aid the development, I had thought about if it is needed to keep the log, given we already have info level log when vtep route changes at run time in `if option.Config.EnableL7Proxy` block and we have ` cilium bpf vtep list` in bugtool, I think we can remove the log code here completely, what do you think?

Reply:
oh, wait, `cilium bpf vtep list` just dump the vtep map list, not vtep route existed in host for debugging purpose, the goal here  is to check if the host has vtep route matches vtep map list for debugging purpose if user ran into issue for some reason

Reply:
I just checked bugtool again, bugtool actually dumps all host route including vtep route, so it should be ok to remove the log since it is for debug purpose, right?

Reply:
ok, I am going to go ahead to remove the log, let me know otherwise :-)